blog/2022-07-05-paralus-quickstart-gke/index.md

```diff
@@ -48,13 +49,13 @@ After the cluster is created, start the cluster and connect to it. You can conne
 
   ```bash
    helm install myrelease paralus/ztka \
-    -f https://raw.githubusercontent.com/paralus/helm-charts/main/examples/values.generic.yaml \
+    -f https://raw.githubusercontent.com/paralus/helm-charts/main/examples/values-dev-generic.yaml \
```

Comment:
```suggestion
    -f https://raw.githubusercontent.com/paralus/helm-charts/main/examples/values.dev-generic.yaml \
```

You might wanna also change the other references to `values.generic.yaml` and `values.eks.yaml`. Those don't exist anymore.

Reply:
Yes, if this looks good, will replicate it for other blog posts as applicable.